examples/with_winit/src/simple_text.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 The piet-gpu authors.
+// Copyright 2022 The vello authors.
```

Comment:
Our copyright headers are a bit of a mess. They just add a pile of cruft to every file, for apparently minimal gain.

I'd be inclined to remove them entirely, following rustc's lead - https://github.com/rust-lang/rust/pull/57108

This is an improvement though.

Reply:
I've kept it as 'vello authors' for now then.